stdpopsim/cli.py

```diff
@@ -149,6 +149,44 @@ def __call__(self, parser, namespace, values, option_string=None):
         parser.exit()
 
 
+def get_species_help(species_id):
+    """
+    Generate help text for the given species with some of the species attributes
+    that are not covered by the other helps. Does not accept any additional
+    options.
+    """
+    species = stdpopsim.get_species(species_id)
+    species_text = f"\nDefault population parameters for {species.name}:\n"
+
+    # Add the default generation times and citation(s)
+    indent = " " * 4
+    wrapper = textwrap.TextWrapper(initial_indent=indent, subsequent_indent=indent)
+    species_text += f"Generation time: {species.generation_time}\n"
+    for gtc in species.generation_time_citations:
+        species_text += wrapper.fill(textwrap.dedent("- " + str(gtc)))
+        species_text += "\n"
+
+    # Add the default population size and citation(s)
+    species_text += f"Population size: {species.population_size}\n"
+    for psc in species.population_size_citations:
+        species_text += wrapper.fill(textwrap.dedent("- " + str(psc)))
+        species_text += "\n"
+
+    # Default mutation rate
+    species_text += f"Mutation rate: {species.genome.mean_mutation_rate}\n"
+    # for mrc in species.genome.mutation_rate_citations:
```

Comment:
I don't think we need citations here in the help --- we only need to write these out afterwards so that users can cite the literature relevant to the simulations they've actually run. Citations in the help text don't help the user in any way, I think.

Reply:
ok